e2e/_suites/fleet/services.go

```diff
@@ -421,8 +421,8 @@ func newDockerInstaller(ubi8 bool, version string) (ElasticAgentInstaller, error
 
 // newTarInstaller returns an instance of the Debian installer for a specific version
 func newTarInstaller(image string, tag string, version string) (ElasticAgentInstaller, error) {
-	image = image + "-systemd" // we want to consume systemd boxes
-	service := image
+	dockerImage := image + "-systemd" // we want to consume systemd boxes
```

Comment:
This variable will be the one holding the docker image name, that's why we replaced all occurrences with `dockerImage`. On the other hand, the `image` variable will hold the _flavour_: centos or debian.